decidim-proposals/app/views/decidim/proposals/admin/proposals/update_category.js.erb

```diff
@@ -1,15 +1,15 @@
 <% if flash.now[:error].present? %>
   $("#js-form-recategorize-proposals #category_id").addClass("is-invalid-input")
-  $("<%= escape_javascript(render partial: %q{js-callout}, locals: { css: %q{alert}, text: flash.now[:error] }) %>").appendTo(".callout-wrapper");
+  $("<%= escape_javascript(render partial: %q{decidim/admin/shared/js-callout}, locals: { css: %q{alert}, text: flash.now[:error] }) %>").appendTo(".callout-wrapper");
```

Comment:
Same thing here, can we try to merge `update_category.js.erb` and `update_scope.js.erb`? They seem very similar.